test/Microsoft.AspNetCore.Mvc.Core.Test/Routing/DispatcherUrlHelperTest.cs

```diff
@@ -101,25 +100,36 @@ protected override IServiceProvider CreateServices()
         private List<MatcherEndpoint> GetDefaultEndpoints()
         {
             var endpoints = new List<MatcherEndpoint>();
-            endpoints.Add(new MatcherEndpoint(
-                next => (httpContext) => Task.CompletedTask,
-                "{controller}/{action}/{id}",
-                new { id = "defaultid" },
-                0,
-                EndpointMetadataCollection.Empty,
-                "RouteWithNoName",
-                address: null));
-            endpoints.Add(new MatcherEndpoint(
-                next => (httpContext) => Task.CompletedTask,
-                "named/{controller}/{action}/{id}",
-                new { id = "defaultid" },
-                0,
-                EndpointMetadataCollection.Empty,
-                "RouteWithNoName",
-                new Address("namedroute")));
+            endpoints.Add(CreateEndpoint(null, "home/newaction/{id?}", new { id = "defaultid", controller = "home", action = "newaction" }, 1));
```

Comment:
FYI: These lines mimic the conventional route setup below (which is used in `UrlHelperTest.cs` file which uses conventional routing):
https://github.com/aspnet/Mvc/blob/4dd4e5ef3e11c2f7682e56b72b739fdbf265c619/test/Microsoft.AspNetCore.Mvc.Core.Test/Routing/UrlHelperTest.cs#L91-L121